### PR TITLE
feat: for-native

### PR DESCRIPTION
**Pull request by Rabursky**
perhaps we should remove locks from repo whatsoever? 🤷

### CODE REVIEW

File: js/packages/for-native/src/mixpanel_backend.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+export class MixpanelBackend implements UIBackend {
+    constructor(token: string, options: {[k: string]: string | number | boolean} = {}) {
+        const {
+            optOutTrackingByDefault,
+            trackCrashes,
+            automaticPushTracking,
+            launchOptions,
+        } = options
+        Mixpanel.sharedInstanceWithToken(
```
Comment (michael-donat) [inline js/packages/for-native/src/mixpanel_backend.ts:13]: Is there no way to not have shared instance?

File: js/package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "root",
   "private": true,
   "devDependencies": {
-    "lerna": "^3.22.1"
+    "lerna": "^3.22.1",
+    "rollup": "^2.26.3"
```
Comment (michael-donat) [inline js/package.json:6]: Shouldn be local to package

File: js/packages/for-native/package.json [hunk 3]
```diff
@@ -2,13 +2,35 @@
   "name": "@utilitywarehouse/customer-tracking-for-native",
   "version": "0.4.7",
   "description": "",
-  "main": "index.js",
-  "scripts": {},
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/cjs/index.d.ts",
+  "browser": "dist/umd/tracker.min.js",
```
Comment (michael-donat) [inline js/packages/for-native/package.json:8]: Not needed

**Review by michael-donat — CHANGES_REQUESTED** [review]
Remove index.js

File: js/packages/for-native/src/mixpanel_backend.ts (hunk 1 above)
Comment (Rabursky) [reply js/packages/for-native/src/mixpanel_backend.ts:13]: doesn't look like it even in the native docs...
https://developer.mixpanel.com/docs/ios

File: js/packages/for-native/package.json (hunk 3 above)
Comment (Rabursky) [reply js/packages/for-native/package.json:8]: 🤦

**Pull request merged**